### DISCUSSION TITLE
How to name a tag wrapper

### DISCUSSION THREAD

**Discussion by pankopiotr** [getbem/getbem.github.io#251] [Q&A]
I was wondering whether `a`  tag element which is wrapping other content, is considered a block?
Let's consider this vanilla code:
```
          <div>
            <a href='#'>
              <p>Some random paragraph</p>
              <img src='stylesheets/assets/onion.jpg'>
            </a>
          </div>
```
Code below considers `a` tag as normal element.
```
          <div class="news-box">
            <a class="news-box__link" href='#'>
              <p class="news__content">Some random paragraph</p>
              <img class="news-box__image" src='stylesheets/assets/onion.jpg'>
            </a>
          </div>
```
While this code considers `a` tag as block that wraps `p` and `img`:

```
          <div class="news-box">
            <a class="news" href='#'>
              <p class="news__content">Some random paragraph</p>
              <img class="news-__image" src='stylesheets/assets/onion.jpg'>
            </a>
          </div>
```

Which approach is better/valid and why?

**Comment by tadatuta** [comment]
Both variants are valid.
Here's the way to decide if an entity is a block or an element: https://en.bem.info/methodology/quick-start/#should-i-create-a-block-or-an-element

**Comment by pankopiotr** [comment]
Thanks a lot.
You can close this issue :)
